app/slashcommands/command_share.go

```diff
@@ -278,17 +285,29 @@ func (sp *ShareProvider) doStatus(a *app.App, args *model.CommandArgs, margs map
 
 	fmt.Fprintf(&sb, "Status for channel Id `%s`\n\n", statuses[0].ChannelId)
 
-	fmt.Fprintf(&sb, "| Remote | Host | Description | ReadOnly | InviteAccepted | Online | Token |\n")
-	fmt.Fprintf(&sb, "| ------ | ---- | ----------- | -------- | -------------- | ------ | ----- |\n")
+	fmt.Fprintf(&sb, "| Remote | SiteURL | Description | ReadOnly | InviteAccepted | Online | Last Sync | \n")
+	fmt.Fprintf(&sb, "| ------ | ------- | ----------- | -------- | -------------- | ------ | --------- | \n")
 
 	for _, status := range statuses {
 		online := ":white_check_mark:"
 		if !isOnline(status.LastPingAt) {
 			online = ":skull_and_crossbones:"
 		}
-		fmt.Fprintf(&sb, "| %s | %s:%d | %s | %t | %t | %s | %s |\n",
-			status.ClusterName, status.Hostname, status.Port, status.Description,
-			status.ReadOnly, status.IsInviteAccepted, online, status.Token)
+
+		lastSync := formatTimestamp(time.Unix(0, status.LastSyncAt*int64(time.Millisecond)))
+
+		fmt.Fprintf(&sb, "| %s | %s | %s | %t | %t | %s | %s |\n",
+			status.DisplayName, status.SiteURL, status.Description,
+			status.ReadOnly, status.IsInviteAccepted, online, lastSync)
 	}
 	return responsef(sb.String())
 }
+
+func (sp *ShareProvider) doSync(a *app.App, args *model.CommandArgs, margs map[string]string) *model.CommandResponse {
+	// start sending updates to remote clusters.
+	a.Srv().GetSharedChannelSyncService().NotifyChannelChanged(args.ChannelId)
```

Comment:
My understanding is that this command will be replaced by the automatic sync notifier which I am working on. I will defer to @wiggin77 for a sanity check.

Reply:
This command was created to facilitate testing before the automatic triggers are available.  I think I'll leave the command in just in case manual sync is the work around for some issue.

Reply:
Understood. Yes, I was referring to the "manual sync" case which doesn't necessarily mean something has changed locally.

Reply:
If I'm following correctly... I would like the manual sync to force a sync even if nothing changed.  Even if all we get on the remote side is a log entry saying it got an empty sync message.